index.js

```diff
@@ -86,6 +86,17 @@ const sendResponse = ( req, settings, fn ) => {
 };
 
 /**
+ * Returns `true` if `v` is a File Form Data, `false` otherwise.
+ *
+ * @param {Mixed} v - instance to analize
+ * @return {Boolean} `true` if `v` is a DOM File instance
+ * @private
+ */
+function isFile( v ) {
+	return v instanceof Object && v.fileContents instanceof Blob;
```

Comment:
Is `fileContents` a Calypso-specific convention or a standard shape for a File-like object?

Also looking at equivalent logic in proxy request:

https://github.com/Automattic/wpcom-proxy-request/blob/912fdb1/index.js#L223-L232